### PR TITLE
feat: add spanish locale, fix: sidebar width (again)

### PR DESCRIPTION
**Pull request by MicrowavedToast21**
Added spanish translation

### CODE REVIEW

File: src/main/resources/assets/minecraft/lang/spanish_spain.json
Comment (PassiHD2004) [inline src/main/resources/assets/minecraft/lang/spanish_spain.json:1]: rename file to es_es.json (should be correct). Test your changes, see if it works

File: src/main/resources/assets/minecraft/lang/en_us.json
Comment (PassiHD2004) [inline src/main/resources/assets/minecraft/lang/en_us.json:1]: re-add english locale? why was it even removed?

**Comment by PassiHD2004** [comment]
now you removed the spanish translations?

File: src/main/resources/assets/minecraft/lang/spanish_spain.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "foxclient.config.uwufy": "UwUficar",
+
+  "foxclient.config.custommainmenu": "Menu principal",
+  "foxclient.config.custompausemenu": "Menu de pausa",
+
+  "foxclient.config.discordenabled": "DiscordRPC - activado",
+  "foxclient.config.discordshowip": "DiscordRPC - Mostrar IP",
+  "foxclient.config.discordshowplayer": "DiscordRPC - Mostrar un jugador",
+
+  "foxclient.config.toggle.on": "Encendido",
```
[the remark this answers is not in the document]
Comment (PassiHD2004) [reply src/main/resources/assets/minecraft/lang/spanish_spain.json]: ah ok

**Comment by PassiHD2004** [comment]
> does it work now?

no, it does not. Spanish locale is MISSING

**Comment by PassiHD2004** [comment]
PR is ready to merge. Flustix or rooot has to merge it tho, since i dont have permission for the main branch

**Comment by RoootTheFox** [comment]
i won't merge this as is - you are modifying the en_us locale.
can you fully remove the commits that do that and commit *only* the spanish locale file? (eg by force pushing)

**Comment by RoootTheFox** [comment]
nevermind me, im blind (it still showed the old PR state), this seems ready to be merged.

**Review by RoootTheFox — APPROVED** [review]
looks good, lgtm

**Comment by RoootTheFox** [comment]
gh won't let me merge this

**Pull request merged**